### ISSUE TITLE
Quick links menu header in the topnav dropdown menu breaks keyboard nav

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#236]
In the drop-down menu from the hamburger icon, the Quick links header is like this:

```
<li role="presentation" class="menu-header disabled">
    <a role="menuitem">Quick links</a>
  </li>
```

which has the effect of stopping arrow-key nav, and it shows up in the links list. I think it should be like this:

```
<li role="presentation" class="dropdown-header">Quick links</li>
```

i.e. not disabled, and no <a> tag.
